### PR TITLE
texlive: revision bump (icu4c 71.1)

### PR DESCRIPTION
**Pull request by carlocab**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

I split this off from #98809 so that the `texlive` bump can be merged ASAP after the `icu4c` bump is merged. This will help us minimise breakage for users.

### CODE REVIEW

**Comment by carlocab** [comment]
Actually, I should probably squash the two commits so that the `icu4c` changes are dropped when @BrewTestBot cherry-picks this PR, and we don't need to wait for a rebase here.

File: Formula/texlive.rb [hunk 1]
```diff
@@ -108,6 +108,11 @@ class Texlive < Formula
     sha256 "372b2b07b1f7d1dd12766cfc7f6656e22c34a5a20d03c1fe80510129361a3f16"
   end
 
+  resource "Pygments" do
+    url "https://files.pythonhosted.org/packages/94/9c/cb656d06950268155f46d4f6ce25d7ffc51a0da47eadf1b164bbf23b718b/Pygments-2.13.0.tar.gz"
+    sha256 "56a8508ae95f98e2b9bdf93a6be5ae3f7d8af858b43e02c5a2ff083726be40c1"
+  end
```
Comment (carlocab) [inline Formula/texlive.rb:114]: Just moved this from below to put all the non-CPAN resources in one place (for `brew update-perl-resources`).

**Comment by carlocab** [comment]
For ease of auditing, the checksums for the bottles built in this PR are [here](https://github.com/Homebrew/homebrew-core/actions/runs/2970482038). This should match what's in the bottle commit.

**Review by Bo98 — DISMISSED** [review]

**Comment by carlocab** [comment]
This was me, despite the unverified commits. Pushed from a machine I use with SSH, and getting commit signatures to work on it when headless is really annoying.

**Comment by carlocab** [comment]
Also, dropped the `revision` for `node@10` -- that bottle was not built because it is disabled.

**Comment by carlocab** [comment]
```
Error: You are trying to merge a pull request that conflicts with a long running build in:
```
Oops

**Pull request closed without merging**